examples/tutorials/colabs/replay_tutorial.ipynb

```diff
@@ -334,6 +334,10 @@
     "chefcan_3 = rigid_obj_mgr.add_object_by_template_handle(chefcan_template_handle)\n",
     "chefcan_3.translation = [2.4, -0.64, -0.28]\n",
     "\n",
+    "# scale some objects\n",
+    "# TODO: this doesn't work because replay is only tracking the MotionState, not the scaling SceneNode further in the tree. Needs a refactor.\n",
```

Comment:
Gfx-replay support for dynamic scaling would be a small change to `Recorder::getInstanceState` and `RenderAssetInstanceState`. If we should decide to move forward with merging this, can you make this change as part of this PR? (So that we avoid incurring this tech debt.)

See also `GfxReplayTest::testSimulatorIntegration` for C++ unit-testing.